python/desc/sims_ci_pipe/psf_whisker_plot.py

```diff
@@ -0,0 +1,171 @@
+"""
+Module to compute visit-level PSF whisker plots.
+"""
+import itertools
+import numpy as np
+from scipy.interpolate import griddata
+import matplotlib.pyplot as plt
+import lsst.geom as lsst_geom
+from .ellipticity_distributions import get_point_sources
+
+
+__all__ = ['get_e_components', 'get_sky_coords', 'psf_whisker_plot']
+
+
+def get_e_components(ixx, iyy, ixy):
+    """
+    Compute ellipticity components from second moments.
+    """
+    e1 = (ixx - iyy)/(ixx + iyy)
```

Comment:
This is the distortion definition (the same used in the LSST-SRD eqs 10-11). However it might be good to also include the "shear" definition (for example with an option):

In that case:
```
detQ = I_psf_xx*I_psf_yy-I_psf_xy**2
e_psf = (I_psf_xx-I_psf_yy+2j*I_psf_xy)/(I_psf_xx+I_psf_yy+2*detQ**0.5) 
e1 = np.real(e_psf)
e2 = np.imag(e_psf)
```

Reply:
Thanks, I've added this option.